README.md

```diff
@@ -1,23 +1,79 @@
+we've learned about text formatting and displaying images.
 
-## Tasks for Local Environment Users
+## Objectives
 
-1. Fork this repository.
-2. Clone your fork locally.
-3. `cd` into the local repo you just cloned.
+- Practice applying header, paragraph, image and span tags
 
-## Process 
+## The Mock Up
 
-1. Take a look at the <a href="https://s3.amazonaws.com/learn-verified/MOCKUP.jpg" target="_blank">MOCKUP.jpg</a> image.
-2. Use the comments in `index.html` to create the elements necessary to recreate the album cover shown in <a href="https://s3.amazonaws.com/learn-verified/MOCKUP.jpg" target="_blank">MOCKUP.jpg</a>. Also use `SOURCE.md` to copy and paste all necessary filler text.
-3. Open up `index.html` in the browser as well for a side by side comparison with <a href="https://s3.amazonaws.com/learn-verified/MOCKUP.jpg" target="_blank">MOCKUP.jpg</a> as you work. To open `index.html` from the IDE, follow [these steps](http://help.learn.co/the-learn-ide/common-ide-questions/viewing-html-pages-in-the-learn-ide).
+Pharrell isn't quite happy, he needs some help coding the album cover for his
+hit song "Happy." His dev chops aren't quite as skilled as yours, so he's hired
+you to take care of business.
 
-***NOTE***: Clicking the blue `Open` button on the Learn.co lesson page or running the `learn open` command in your terminal will automatically fork and clone the repository and change into the appropriate directory.
+Pharrell did provide us with a mock up of what he would like to see for the
+design of the album cover:
+
+![Happy Album Cover](./MOCKUP.jpg)
+
+In the industry, frontend developers are typically given specs just
+like this from the product team and then asked to write the code to make a
+website look just like the spec. Today, you're the developer and Pharrell is
+the product team.
+
+### SOURCE.md
+
+Pharrell has provided us with the text for the album and all the legal
+mumbo-jumbo to give credit where credit is due. You can find that text in
+`SOURCE.md`. Again, this is a typical industry standard workflow, where the
+developers are given copy for the site from the product team.
+
+Each line of text in this file corresponds to a note in `index.html` that will
+indicate where you'll need to add this information. It will be up to you to pick
+which HTML tags to use.
+
+### Getting To Work
+
+Write your code in `index.html`. Your job is to make sure the text from
+`SOURCE.md` makes it into `index.html` surrounded by the appropriate HTML tags.
+
+The files you need are provided in the 'images' folder, but you'll have to write
+the correct _relative paths_ in order for them to display correctly.
+
+You may not have encountered the `span` tag before, so if you are curious, take
+a moment to [read up on some reference material] to better understand its use.
+
+Run `httpserver` and copy the IP address it provides to see the album cover as
+you build it.
```

Comment:
We should update this with the appropriate directions for both the IDE and local environments.